app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RestaurantsController < ApplicationController
+
+  def new
+    @restaurants = Restaurant.new
+  end
+
+  def create
+    @restaurants = Restaurant.new restaurant_params
```

Comment:
Same here. The plural form should be used for a collection. I.e. In the index method (if you create one later)